src/components/App.js

```diff
@@ -9,61 +9,69 @@ import yaml from 'js-yaml';
 let githubClient;
 
 export default class App extends Component {
-  constructor(props) {
```

Comment:
Ok. I downloaded [Airbnb Style Preset](https://github.com/jscs-dev/node-jscs/blob/master/presets/airbnb.json) and imported it into webstorm. It'll reformat code according to their settings.